bridges/relays/lib-substrate-relay/src/messages/mod.rs

```diff
@@ -416,7 +416,7 @@ where
 				"Prepared {} -> {} messages delivery call. Weight: {}/{}, size: {}/{}",
 				P::SourceChain::NAME,
 				P::TargetChain::NAME,
-				call.get_dispatch_info().weight,
+				call.get_dispatch_info().call_weight,
```

Comment:
maybe total_weight is more relevant?

Reply:
It doesn't matter because we call `get_dispatch_info` on a call which will always return an `extension_weight` of `0`.